cmd/envtool/envtool.go

```diff
@@ -361,6 +363,18 @@ func read(w io.Writer, paths ...string) error {
 	return nil
 }
 
+// printVersion will print out version omitting leading v
+func printVersion(w io.Writer, file string) error {
+	b, err := os.ReadFile(file)
+	if err != nil {
+		return err
+	}
+
+	fmt.Fprintf(w, string(b)[1:])
```

Comment:
I suggest before trimming the data, we check that the string in the file has prefix `v`, and trim it only in this case. 
Currently, our versions always have the prefix `v`, but overall it looks like a nice extra check. E.g. if the prefix is `v` we trim it and return the rest, otherwise we return data as is.

Reply:
Done!